test/testdata/compiler/proc_arity.rb

```diff
@@ -32,3 +41,4 @@ def test_case(name, p)
 test_case("1 required, 1 defaulted, repeated", lambda {|a, b=10, *c|})
 test_case("2 required, 1 defaulted, repeated", lambda {|a, b, c=10, *d|})
 test_case("2 required, 2 defaulted, repeated", lambda {|a, b, c=10, d=nil, *e|})
+test_case("2 required, 2 defaulted, repeated, break", lambda {|a, b, c=10, d=nil, *e| break})
```

Comment:
Comment in the same spirit as above: we should try to have some required/defaulted kwargs in here too?